src/room/useLoadGroupCall.ts

```diff
@@ -117,9 +117,9 @@ export class CallTerminatedMessage extends Error {
 }
 
 export const useLoadGroupCall = (
-  client: MatrixClient,
-  roomIdOrAlias: string,
+  roomIdOrAlias: string | null,
   viaServers: string[],
+  client?: MatrixClient,
```

Comment:
```suggestion
  client: MatrixClient | undefined,
```

to remind consumers that they do need to pass a client eventually